fund_my_watcard/watcard

```diff
@@ -68,8 +69,11 @@ class MyWatCard:
 
     def add_fund(self, amount):
         try:
-            with Browser("chrome", **{"executable_path": ChromeDriverManager().install()}, incognito=True,
-                         headless=True) as browser:
+            chrome_options = webdriver.ChromeOptions()
+            chrome_options.add_argument('--headless')
+            chrome_options.add_argument('--no-sandbox')
```

Comment:
Why this option is necessary for us?

Reply:
Please read: https://sites.google.com/a/chromium.org/chromedriver/help/chrome-doesn-t-start